src/initialization.js

```diff
@@ -5,7 +5,8 @@ var helpers = require('./helpers');
 var initialization = {
     name: 'Optimizely',
     moduleId: 54,
-    initForwarder: function(settings, testMode, userAttributes, userIdentities, processEvent, eventQueue, isInitialized) {
+    initForwarder: function(settings, testMode, userAttributes, userIdentities, processEvent, eventQueue, isInitialized,
+                            common, appVersion, appName, customFlags, clientId) {
```

Comment:
small nit - looks like this popped onto line 9 somehow, move up to be in line with line 8